.github/workflows/call-plugin-workflow.yml

```diff
@@ -1,5 +1,9 @@
 name: Run plugin workflow
 
+concurrency:
```

Comment:
Can you move this into our reusable workflow?
This would make sense as this would allow use to reuse code instead of pasting this in every workflow.

Or did you find out why it does not work in the reusable workflow,m